src/styles/components/_content-card.scss

```diff
@@ -158,11 +157,13 @@
   &::before
   {
     content: "";
+    display: block;
     position: absolute;
     top: 0;
     right: 0;
     bottom: 0;
     left: 0;
+    z-index: 100;
```

Comment:
Let's look at reducing this value. We should compare with primary nav items/panels and modal to make sure we can't activate product card links accidentally.